### PR TITLE
Add 'download-atom-shell-chromedriver' task

### PR DESCRIPTION
**Pull request by maxbrunsfeld**
Refs atom/atom#5404

### CODE REVIEW

File: tasks/download-atom-shell-task.coffee [hunk 1]
```diff
@@ -98,71 +102,117 @@ module.exports = (grunt) ->
...
+      return done()
+
+    # Request the assets.
+    github = new GitHub({repo: 'atom/atom-shell', token})
+    github.getReleases tag_name: versionWithChromedriver, (error, releases) ->
+      unless releases?.length > 0
+        grunt.log.error "Cannot find atom-shell #{versionWithChromedriver} from GitHub", error
+        return done false
+
+      assetNameRegex = new RegExp("chromedriver-.*-#{process.platform}-#{getArch()}")
```
Comment (kevinsawicki) [inline tasks/download-atom-shell-task.coffee]: Super minor, but the CoffeeScript interpolated regexes using `///` makes things a litter terser.

```coffee
assetNameRegex = ///chromedriver-.*-#{process.platform}-#{getArch()}"///
```

Comment (maxbrunsfeld) [inline tasks/download-atom-shell-task.coffee]: Nice :sparkles:

**Comment by kevinsawicki** [comment]
Looks good, mind mentioning it in the README.md?

**Comment by maxbrunsfeld** [comment]
Good idea. I also removed a chunk of the README where it showed the same code snippet in both coffee-script and plain javascript; seemed unnecessary in this context. Seem ok?

**Comment by kevinsawicki** [comment]
> seemed unnecessary in this context. Seem ok?

I think having both is nice just so people can easily copy/pasta for either Gruntfile language and get going.

**Comment by maxbrunsfeld** [comment]
:ok_hand:

**Pull request merged**